apps/backend/src/deposits/cash-deposit.service.ts

```diff
@@ -67,32 +67,30 @@ export class CashDepositService {
         status: In(depositStatus),
       },
       order: {
-        deposit_amt_cdn: 'ASC',
+        deposit_amt_cdn: 'DESC',
       },
     });
   }
-
-  async findCashDepositDatesByLocation(
+  /**
+   * Find all cash deposits for a particular location
```

Comment:
Amend comment? All cash deposit **dates**

Reply:
oops, thank you!